spec/services/users/set_attributes_service_integration_spec.rb

```diff
@@ -40,16 +40,6 @@
 
   subject { instance.call(params) }
 
-  context "with a boolean castable preference" do
```

Comment:
This testcase could be changed to test another boolean preference, for example `warn_on_leaving_unsaved`